ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -82,6 +82,13 @@ then
 
 	# Internal SPI bus ICM-20608-G
 	mpu6000 -T 20608 start
+	
+	if param compare SYS_COMPANION 1500000
+	then
+		if amov_imu start -X 3 -T 1 -I 2 -A -P 255
```

Comment:
This doesn't need to be wrapped in a conditional.
Do you want to add a comment explaining these options? Could some of them be defaults instead?

Reply:
This command is aimed to start the external IMU by default from the TELEM 2, which is the most common port for all autopilots.